### DISCUSSION TITLE
Integrazioni Matter e Cloudflare insieme

### DISCUSSION THREAD

**Discussion by ziocharly** [indomus/forum#2668] [HUB: Home Assistant]
Ho fatto qualche prova con ESP Zero Code ed il nuovo procollo Matter.
Nessun problema e tutto funziona bene, salvo un'incompatibilità con l'integrazione CloudFlare.
Come noto infatti il protocollo Matter necessità di IP V.6 abilitato.

Sfortunatamente, con IP V.6 abilitato in HA, l'integrazione CloudFlare va in errore
cercando di aggiornare il record dell'IP dinamico assegnato.
In rete é confermato che il problema scompare disabilitando IP V.6.
L'ho potuto praticamente constatare anch'io...

Qualcuno ha già trovato  una qualche soluzione ?

**Comment by KC-inDomus** [comment]
parli di Home Assisrtant OS/Supervised presumo, perchè con Core@Docker, IPv6 attivo e Cloudflare@Docker, nessun problema.

**Comment by ziocharly** [comment]
Esatto, HA OS 11.0 su RBPI4, questo é il problema:
https://github.com/home-assistant/core/issues/84959
 
...Should I just disable IPv6 in HA - it seems that everywhere I turn, no-one seems to be prepared for its use...

**Comment by ziocharly** [comment]
Spero in qualche aggiornamento perché CloudFlare é indispensabile per l'update dell'IP (Vodafone cambia 2-3 volte a settimana)
e Matter é un protocollo emergente, da tenere d'occhio e sperimentare...

https://github.com/home-assistant/core/issues/84959#issuecomment-1371184969

**Comment by ziocharly** [comment]
Possibile che sia l'unico ad aver riscontrato questo problema...
